WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginSiteAddressFragment.java

```diff
@@ -0,0 +1,376 @@
+    private static final String KEY_IN_PROGRESS = "KEY_IN_PROGRESS";
+    private static final String KEY_REQUESTED_SITE_ADDRESS = "KEY_REQUESTED_SITE_ADDRESS";
+
+    public static final String TAG = "login_email_fragment_tag";
+
+    private static final String XMLRPC_BLOCKED_HELPSHIFT_FAQ_SECTION = "10";
+    private static final String XMLRPC_BLOCKED_HELPSHIFT_FAQ_ID = "102";
+
+    private static final String MISSING_XMLRPC_METHOD_HELPSHIFT_FAQ_SECTION = "10";
+    private static final String MISSING_XMLRPC_METHOD_HELPSHIFT_FAQ_ID = "11";
+
+    private static final String NO_SITE_HELPSHIFT_FAQ_SECTION = "10";
+    private static final String NO_SITE_HELPSHIFT_FAQ_ID = "2"; //using the same as in INVALID URL
+
+    private TextInputLayout mSiteAddressTextLayout;
+    private EditText mSiteAddressEditText;
+    private Button mNextButton;
+    private ProgressDialog mProgressDialog;
+
+    private LoginListener mLoginListener;
+
+    private boolean mInProgress;
+    private String mRequestedSiteAddress;
+
+    @Inject Dispatcher mDispatcher;
+    @Inject HTTPAuthManager mHTTPAuthManager;
+    @Inject MemorizingTrustManager mMemorizingTrustManager;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((WordPress) getActivity().getApplication()).component().inject(this);
+
+        setHasOptionsMenu(true);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        ViewGroup rootView = (ViewGroup) inflater.inflate(R.layout.login_site_address_screen, container, false);
+
+        mSiteAddressEditText = (EditText) rootView.findViewById(R.id.login_site_address);
+        mSiteAddressEditText.addTextChangedListener(this);
+
+        mSiteAddressTextLayout = (TextInputLayout) rootView.findViewById(R.id.login_site_address_layout);
+
+        mNextButton = (Button) rootView.findViewById(R.id.login_next_button);
+        mNextButton.setOnClickListener(new OnClickListener() {
+            public void onClick(View v) {
+                discover();
+            }
+        });
+
+        mSiteAddressEditText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
+                if (event != null
+                        && event.getAction() == KeyEvent.ACTION_UP
+                        && event.getKeyCode() == KeyEvent.KEYCODE_ENTER) {
+                    discover();
+                }
+
+                // always consume the event so the focus stays in the EditText
+                return true;
+            }
+        });
+
+        rootView.findViewById(R.id.login_site_address_help).setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (mLoginListener != null) {
```

Comment:
Hah, good catch! That was a relic from a previous revision of the code. Removed with 39562a0.